lib/common/xml.c

```diff
@@ -3895,7 +3923,8 @@ crm_xml_dump(xmlNode * data, int options, char **buffer, int *offset, int *max,
             dump_xml_element(data, options, buffer, offset, max, depth);
             break;
         case XML_TEXT_NODE:
-            /* Ignore */
+            /* Handle the text node*/
+            dump_xml_text(data, options, buffer, offset, max, depth);
```

Comment:
I think XML_TEXT_NODE is ignored on purple in here.  We'd better introduce an xml_log_option like xml_log_option_text,  and a dump_xml_formatted_with_text() function to achieve this without changing the default behaviors.